server/models/Wallet.spec.js

```diff
@@ -281,7 +281,7 @@ describe("Wallet", () => {
         target_entity_id: wallet.getId(),
       }]);//no relationship
       const result = await wallet.hasTrust(
-        TrustRelationship.ENTITY_TRUST_TYPE.send,
+        TrustRelationship.ENTITY_TRUST_REQUEST_TYPE.receive,
```

Comment:
@dadiorchen As a result of my change on hasFix method referencing the trust type parameter now, I also think this test should expect a "receive" request type. Since wallet2 (actor, receiver) is sending a receive request to wallet (target, sender).